### DISCUSSION TITLE
关于在线检谱显示结果UI的建议

### DISCUSSION THREAD

**Discussion by zglzs** [Equim-chan/mjai-reviewer#97] [Ideas]
你好，感谢您的作品
1.现在微差的牌也标红，还要点开详情看，比较难找有用的信息。建议根据分数显示不同的颜色，例如微差就标黄，严重选择错误的标橙色红色，或者直接显示player和mortal discard的分数。
2.建议加一个功能：和mortal选择最不一致的几次选择，也就是所谓“鸡打”，做一个summary放前面。这样我觉得会更方便学习。

**Comment by Equim-chan** [comment]
1. 这是故意这么设计的。分数本身是和具体 model 以及温度直接相关的，换一个 model 体系就不一样了，所以没法用一个固定的标准去衡量“微差”之类的情况，需要用户自己去做判断。
2. 基于类似的理由，目前不太倾向于实现这个设计。

**Comment by chucklu** [comment]
有人做了一个叫做MortalPlus的插件，可以显示恶手 @zglzs ,和Detail里面的差距很大的打法
